test/performance/logs/parameters.yml

```diff
@@ -0,0 +1,72 @@
+receivers: "logs"
```

Comment:
The `parameters.yml` is the one who has all the information and instruct the validator to generate the logs (e.g receivers specify we would generate logs and the number of monitor logs would be 1000 - `number_monitored_logs`). It also includes all the metrics the [performance wants to capture](https://github.com/aws/amazon-cloudwatch-agent-test/pull/151/files#diff-4182821602cec8db84c8b29427a65c5a63beff4c19bcf93e305966bed05a721eR20-R72)